.gitlab-ci.yml

```diff
@@ -17,69 +16,45 @@ stages:
 check_dependencies:
   stage: check_dependencies
   script:
-    - git ls-remote https://github.com/pcengines/release_manifests.git | grep ${CI_COMMIT_REF_NAME}
+    - git ls-remote https://github.com/pcengines/coreboot.git | grep ${CI_COMMIT_REF_NAME}
   only:
-    - tags
-
-build_crosscompiler:
-  stage: build_crosscompiler
-  tags:
-    - docker
-  script:
-    - test -e ${CI_PROJECT_DIR}/apu2_fw_rel/apu2/coreboot/util/crossgcc/xgcc || { make crossgcc-i386 CPUS=$(nproc) &&  mkdir -p ${CI_PROJECT_DIR}/apu2_fw_rel/apu2/coreboot/util/crossgcc/ && mv -f ${CI_PROJECT_DIR}/util/crossgcc/xgcc ${CI_PROJECT_DIR}/apu2_fw_rel/apu2/coreboot/util/crossgcc/ ; }
-  cache:
-    key: apu_legacy
-    paths:
-      - ${CI_PROJECT_DIR}/apu2_fw_rel/apu2/coreboot/util/crossgcc/xgcc
-  only:
-    - tags
+    - tags 
     - web
```

Comment:
`- web` option for check_dependencies stage is obsolete when building from branches. We trigger pipeline from given coreboot branch so it is obvious that the branch exists. However if a branch exists only on gitlab mirror, the pipeline should fail (this situation is likely because gitlab's mirror even keeps branches that has been deleted from github).

Reply:
@miczyg1 are you suggesting removing that?

Reply:
@pietrushnic IMHO it is not needed for branches